core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -225,12 +225,12 @@ long getNewBlockIdForFile(AlluxioURI path)
   Map<String, MountPointInfo>  getMountTable();
 
   /**
-   * Gets the mount point information of an Alluxio path.
+   * Gets the mount point information of an Alluxio path for display purpose.
    *
    * @param path an Alluxio path which must be a mount point
    * @return the mount point information
    */
-  MountPointInfo getMountPointInfo(AlluxioURI path) throws InvalidPathException;
```

Comment:
instead of rename the internal API in a patch release, can we keep its original name "`getMountPointInfo`"?

Reply:
I explicitly renamed it so that others won't accidentally use it and expect to extract raw information such as credentials. If you are worrying about compatibility, I can fork a copy of the original method just in case someone depends on it in their application.

Reply:
yeah, it will be good to keep the original API

Reply:
Done.